### ISSUE TITLE
[11] CSS is used to include images that convey important information

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#284]
This issue was identified as part of the accessibility testing performed on the DTA website Jan 2017: https://drive.google.com/open?id=0B3SXZ9dr83v0Y2cwV21HdUZ1STQ

The DTA logo in the header is included using CSS.

Only images that are decorative should be included using CSS. When high contrast mode is on, CSS background images disappear in some web browsers. Thus, high contrast users do not know there is an image there at all.

This has particular impact on screens where the logo is the only way to return to the DTA homepage.

![image](https://cloud.githubusercontent.com/assets/2324569/23447444/231c4e3c-fe9f-11e6-9404-988c4f729eec.png)

Image: DTA website header with high contrast mode on (screenshot taken in IE). The DTA logo is not visible.

**Recommendation**

Include meaningful or informative images using the img element with an alt attribute that describes the purpose of the image.

**Comment by bjgordon** [comment]
Easy fix is to start using modernizer to make it easier to fallback from svg to png.
